### ISSUE TITLE
Mew Grid View extension

### ISSUE THREAD

**Issue by Sandracjsilva** [Fugiman/google-meet-grid-view#182]
**Is your feature request related to a problem? Please describe.**
Yes, the new grid view extension it id not very effective. 
1. The names of the participants do not appear;
2. The grid does not occupy the whole available space;
3. The rectangles with the videos of the participants are tiny;
4. The presentation will nor override the grid view, that means it becomes a tiny rectangle in th grid, which will get bigger if we disable grid vie. It will not be a big issue if you are working with adults but with small children it is a big issue and an unnecessary one, in my point of view.
5. If someone leaves it leaves a black/blank rectangle.
6.Cannot see where the noise is coming from.

**Describe the solution you'd like**
A grid view where we can see the names of the participants.
When someone is presenting the presentation becomes bigger than the rest.
Where we can see the sound meter, so we can recognise where the noise is coming from and say what it has become the teachers new known features "please mute your microphone!"
Basically, like it was before. There was nothing wrong with the grid view before.

Today it really affected my lessons, I had a system already in place and today could not use it.
Thank you,
Sandra

**Comment by Fugiman** [comment]
Turn off "Screen Capture Mode"
